fhevm/tee_arithmetic.go

```diff
@@ -110,6 +110,18 @@ func teeMulRun(environment EVMEnvironment, caller common.Address, addr common.Ad
 	})
 }
 
+func teeDivRun(environment EVMEnvironment, caller common.Address, addr common.Address, input []byte, readOnly bool, runSpan trace.Span) ([]byte, error) {
+	return doArithmeticOperation("teeMulRun", environment, caller, input, runSpan, func(a uint64, b uint64) uint64 {
+		return (a - a % b) / b
```

Comment:
Why did you choose this instead of simply a/b?

Reply:
Ah. I mistake someting.